### PR TITLE
Consolidate submission details

### PR DESCRIPTION
**Pull request by jabrah**
#539 
#568 
#582 

Changes are focused in `submission/details`

* Remove old sections: repositories, manuscript IDs, deposits, external repositories
* New display section added intended to consolidate information about repositories, repositoryCopies, and deposits. See ticket for specs

* Update status display in Submissions table
* Revert header fonts

### CODE REVIEW

File: app/templates/components/submission-repo-details.hbs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+</style>
+<li>
+  <p class="card-title">
+    <strong>{{repo.name}} ({{#if repo.url}}<a href={{repo.url}}>{{repo.url}}</a>{{/if}})</strong>
+  </p>
+  <p class="card-text">
+    <div class="sub-status">
+      {{#if repoCopy}}
+        {{#if (eq repoCopy.copyStatus "complete")}}
+          <span class="Complete" tooltip-position="left" tooltip="Submission to this repository was accepted and processed that the repository. ID(s) have been assigned to the submitted manuscript">
```
Comment (karenhanson) [inline app/templates/components/submission-repo-details.hbs]: Unfortunately there was a typo in the ticket that got carried through. Could also be shortened to say, "Submission was accepted and processed by the repository. ID(s) have been assigned to the submitted manuscript."

File: app/templates/components/submission-repo-details.hbs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <i class="fas fa-info-circle"></i> Complete
+          </span>
+          {{repocopy-display repoCopy=repoCopy}}
+        {{/if}}
+        {{#if (eq repoCopy.copyStatus "stalled")}}
+          <span class="stalled" tooltip-position="left" tooltip="">
```
Comment (karenhanson) [inline app/templates/components/submission-repo-details.hbs]: this tooltip needs some text

File: app/templates/components/submission-repo-details.hbs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <i class="fas fa-info-circle"></i> Stalled
+          </span>
+        {{/if}}
+      {{else}}
+        {{#if (eq repo.url "https://eric.ed.gov/")}}
+          <span>Deposit into {{repo.name}} was prompted.</span>
+        {{else}}
+          <span class="" tooltip-position="left" tooltip="PASS has received your submission. Your submission has been sent to the target repositories or is in queue to be sent">
```
Comment (karenhanson) [inline app/templates/components/submission-repo-details.hbs]: If this tooltip will only ever be used for submission details page, it should be "sent to the target repository" - singular

**Comment by jabrah** [comment]
Tooltips have been updated. Code should be a bit cleaner.

**Comment by jabrah** [comment]
Header fonts also updated

**Pull request merged**